benchmark/distributed/master.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+const jstp = require('../..');
+
+const yargs = require('yargs');
+const fs = require('fs');
+const path = require('path');
+const parseAddress = require('./parse-address');
+const statistics = require('../statistics');
+
+const args = yargs
+  .option('address', {
+    alias: 'A',
+    describe: 'Address for master server to listen on',
+    type: 'string',
+    default: 'tcp://localhost:3000',
+  })
+  .options('report', {
+    alias: 'R',
+    describe: 'Report interval in ms',
+    type: 'number',
+    default: 10000,
+  })
+  .options('key', {
+    alias: 'K',
+    describe: 'path to file that contains SSL certificate key',
+    type: 'string',
+    coerce:
+      arg => (arg ? fs.readFileSync(path.resolve(__dirname, arg)) : undefined),
+  })
+  .options('cert', {
+    alias: 'C',
+    describe: 'path to file that contains SSL certificate',
+    type: 'string',
+    coerce:
+      arg => (arg ? fs.readFileSync(path.resolve(__dirname, arg)) : undefined),
+  })
+  .help()
+  .alias('help', 'h')
```

Comment:
Well I'm not sure about common practice as all Linux tools I've used have options in both cases (mostly because there are so many of them), but most of the options I use every day are lowercase so I get an impression that those are more common instead. But in any case, you've created this 'tool' so it's up to you to decide on case.